.github/workflows/ansible-test.yml

```diff
@@ -70,14 +87,19 @@ jobs:
         with:
           python-version: 3.6
 
-      - name: Install ansible-base (devel)
-        run: pip install https://github.com/ansible/ansible/archive/devel.tar.gz --disable-pip-version-check
+      - name: Uninstall any existing ansible
+        run: sudo apt-get remove ansible -y
+
+      - name: Install ansible-base (${{ matrix.ansible }})
+        run: sudo pip install https://github.com/ansible/ansible/archive/${{ matrix.ansible }}.tar.gz --disable-pip-version-check
 
       - name: Run integration tests in ${{ matrix.docker_container }} Docker container
         run: ansible-test integration --docker ${{ matrix.docker_container }} -v --color --retry-on-error --continue-on-error --diff --coverage
+        working-directory: ./ansible_collections/community/sops
 
       - name: Generate coverage report.
         run: ansible-test coverage xml -v --requirements --group-by command --group-by version
```

Comment:
@gundalow for ansible-base >= 2.10, another step is needed: https://github.com/ansible-collections/community.general/blob/main/tests/utils/shippable/shippable.sh#L153

Not that urgent though :)